app/components/ViewContainer.js

```diff
@@ -54,21 +71,26 @@ export default class ViewContainer extends Component {
           targetPin={targetPin}
           setTarget={setTarget}
           /> : void 0 }
-      { this.state.view != 'list' ? <Button
-          onPress={this.toggleView.bind(this, 'list')}
-        >
-          List
-        </Button> : void 0 }
-      { this.state.view != 'ar' ? <Button
-          onPress={this.toggleView.bind(this, 'ar')}
-        >
-          AR
-        </Button> : void 0 }
-      { this.state.view != 'map' ? <Button
+        <View style={styles.ViewMenu}>
+        { this.state.view != 'ar' ? <Button
+            style={styles.ViewButton}
+            onPress={this.toggleView.bind(this, 'ar')}
+          >
+            AR
+          </Button> : void 0 }
+        { this.state.view != 'map' ? <Button
+          style={styles.ViewButton}
           onPress={this.toggleView.bind(this, 'map')}
-        >
+          >
           Map
-        </Button> : void 0 }
+          </Button> : void 0 }
```

Comment:
@tiffeli Totally agree. For the View menu, I think bumping the list down is probably the easiest fix. 

As for the `DropNewPinButton`, I think ideally we'd want to hide it from the `List` view altogether, since the concept of "dropping" a pin needs somewhere to drop it. A quick solution for now would be another conditional on the `ViewContainer` component.